lib/classes/CLI.js

```diff
@@ -216,7 +216,7 @@ class CLI {
     this.consoleLog(chalk.yellow.underline('Plugins'));
 
     if (this.loadedPlugins.length) {
-      const sortedPlugins = _.sortBy(this.loadedPlugins, plugin => plugin.constructor.name);
+      const sortedPlugins = this.loadedPlugins.sort(plugin => plugin.constructor.name);
```

Comment:
This won't work, `sort` and `sortBy` work differently (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort). It needs to be replaced to something as:

```
sort((plugin1, plugin2) => plugin1.constructor.name.localeCompare(plugin2.constructor.name));
```

Still, please also double check we maintain the originally intended order.

Reply:
Thanks for pointing out. I only did some manually test with the callback given to the built-in `sort()` function takes 1 parameter, and it worked with the dataset I chose. So I went out and replaced them. I'll take a deeper look at this.